test/java/integration/AndroidEnvoyEngineStartUpTest.java

```diff
@@ -0,0 +1,29 @@
+package test.kotlin.integration;
+
+import android.content.Context;
+import androidx.test.core.app.ApplicationProvider;
+import io.envoyproxy.envoymobile.AndroidEngineBuilder;
+import io.envoyproxy.envoymobile.Engine;
+import io.envoyproxy.envoymobile.engine.AndroidJniLibrary;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.RobolectricTestRunner;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+// NOLINT(namespace-envoy)
```

Comment:
I don't understand why envoy's check still runs here but I was planning on investigating on a separate pr

Reply:
My local format check passes 😂

Reply:
@buildbreaker  You can add the `./test/java ./test/kotlin ./test/objective-c ./test/swift` paths to `no_namespace_envoy` in check_format.sh

Reply:
I added them: https://github.com/envoyproxy/envoy-mobile/blob/5dc1023d28e435038a88e6c560999f3908c5984a/tools/check_format.sh#L18

😢